macaroonbakery/tests/test_client.py

```diff
@@ -25,7 +28,13 @@
 TEST_OP = bakery.Op(entity='test', action='test')
 
 
-class TestClient(TestCase):
+class TestClient(TestWithFixtures):
+    def setUp(self):
+        super(TestClient, self).setUp()
+        # http_proxy would cause requests to talk to the proxy, which is
+        # unlikely to know how to talk to the test server.
+        self.useFixture(EnvironmentVariable('http_proxy'))
```

Comment:
I suspect that HTTP_PROXY is also an issue and should probably be set too.

Reply:
Done, thanks.